README.md

```diff
@@ -243,26 +258,26 @@ wget http://raspberry.amperka.com/web-server.zip -0 web-server.zip
 
 ```python
 from flask import Flask, send_file
-
+ 
 app = Flask('landingPage')
-
+ 
 @app.route('/')
 def index():
     return send_file('landing.html')
-
+ 
 @app.route('/images/<filename>')
 def get_image(filename):
     return send_file('images/'+filename)
-
-app.run(debug=True, port=3000, host='0.0.0.0')
+ 
+app.run(port=3000, host='0.0.0.0')
 ```
 
 Sometimes following steps 8 to 12 you can get console message:  
 `socket.error: [Errno 48] Address already in use`  
 It means that server is still running. In this case use a command:
 
 ```bash
-ps -fA | grep python
+ps -u | grep python
```

Comment:
Why?

Reply:
to lower the quantity of process that show up